.taplo.toml

```diff
@@ -1,11 +1,12 @@
 exclude = [".cargo/config.toml", "target/**"]
+include = ["examples/**/*.toml"]
 
 [formatting]
-align_entries         = true
-allowed_blank_lines   = 1
-reorder_arrays        = true
+align_entries = true
```

Comment:
Why are these entries not aligned anymore now? 😂

Reply:
Hooooly shit, I assumed the only change here was the `include` thing. I'll double check 🤦